packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard-light/dedicatedCloud-dashboard-light.component.js

```diff
@@ -0,0 +1,31 @@
+import template from './dedicatedCloud-dashboard-light.html';
+
+export default {
+  bindings: {
+    currentDrp: '<',
+    currentUser: '<',
+    goToVpnConfiguration: '<',
+    pccType: '<',
+    isLv1Lv2BannerAvailable: '<',
+    productId: '<',
+    currentService: '<',
+    datacenterList: '<',
+    deleteDrp: '<',
+    disableVmwareOption: '<',
+    drpGlobalStatus: '<',
+    editDetails: '<',
+    goToDrp: '<',
+    goToDrpDatacenterSelection: '<',
+    isDrpActionPossible: '<',
+    onTerminate: '<',
+    onMlSubscribe: '<',
+    orderSecurityOption: '<',
+    orderVmwareOption: '<',
+    isMailingListSubscriptionAvailable: '<',
+    setMessage: '<',
+    onUpgradeVersion: '<',
+    associateIpBlockLink: '<',
+    onExecutionDateChange: '<',
+  },
```

Comment:
You can remove unused props for light mode here to ease to identify which features are still available in that case